docs/contributing.rst

```diff
@@ -11,24 +11,23 @@ Contributing to Scrapy
 
 There are many ways to contribute to Scrapy. Here are some of them:
 
-* Blog about Scrapy. Tell the world how you're using Scrapy. This will help
-  newcomers with more examples and will help the Scrapy project to increase its
-  visibility.
-
 * Report bugs and request features in the `issue tracker`_, trying to follow
   the guidelines detailed in `Reporting bugs`_ below.
 
 * Submit patches for new functionalities and/or bug fixes. Please read
   :ref:`writing-patches` and `Submitting patches`_ below for details on how to
   write and submit a patch.
 
+* Blog about Scrapy. Tell the world how you're using Scrapy. This will help
+  newcomers with more examples and will help the Scrapy project to increase its
+  visibility.
```

Comment:
No strong opinion on the order change, although I think the sorting should be from lowest difficulty to highest difficulty, and submitting patches is usually harder than blogging.